src/soter/boringssl/soter_sign_ecdsa.c

```diff
@@ -1,127 +1,133 @@
+                                                      size_t* key_length, bool isprivate)
 {
-    return soter_ec_export_key(ctx,key,key_length, isprivate);
+    return soter_ec_export_key(ctx, key, key_length, isprivate);
 }
 
-soter_status_t soter_sign_update_ecdsa_none_pkcs8(soter_sign_ctx_t* ctx, const void* data, const size_t data_length)
+soter_status_t soter_sign_update_ecdsa_none_pkcs8(soter_sign_ctx_t* ctx, const void* data,
+                                                  const size_t data_length)
 {
-  if(EVP_DigestSignUpdate(ctx->md_ctx, data, data_length)!=1){
-    return SOTER_FAIL;
-  }
-  return SOTER_SUCCESS;
+    if (EVP_DigestSignUpdate(ctx->md_ctx, data, data_length) != 1) {
+        return SOTER_FAIL;
+    }
+    return SOTER_SUCCESS;
 }
 
-soter_status_t soter_sign_final_ecdsa_none_pkcs8(soter_sign_ctx_t* ctx, void* signature, size_t *signature_length)
+soter_status_t soter_sign_final_ecdsa_none_pkcs8(soter_sign_ctx_t* ctx, void* signature,
+                                                 size_t* signature_length)
 {
-  EVP_PKEY *pkey = EVP_PKEY_CTX_get0_pkey(ctx->pkey_ctx);
-  if (!pkey) {
-    return SOTER_INVALID_PARAMETER;
-  }
-  if (EVP_PKEY_type(pkey->type)!=EVP_PKEY_EC) {
-    return SOTER_INVALID_PARAMETER;
-  }
-  /* TODO: need review */
-  if(!signature || (*signature_length)<(size_t)EVP_PKEY_size(pkey)){
-    (*signature_length)=(size_t)EVP_PKEY_size(pkey);
-    return SOTER_BUFFER_TOO_SMALL;
-  }
+    EVP_PKEY* pkey = EVP_PKEY_CTX_get0_pkey(ctx->pkey_ctx);
+    if (!pkey) {
+        return SOTER_INVALID_PARAMETER;
+    }
+    if (EVP_PKEY_type(pkey->type) != EVP_PKEY_EC) {
+        return SOTER_INVALID_PARAMETER;
+    }
+    /* TODO: need review */
+    if (!signature || (*signature_length) < (size_t)EVP_PKEY_size(pkey)) {
+        (*signature_length) = (size_t)EVP_PKEY_size(pkey);
+        return SOTER_BUFFER_TOO_SMALL;
+    }
 
-  if(EVP_DigestSignFinal(ctx->md_ctx, signature, signature_length)!=1){
-    return SOTER_INVALID_SIGNATURE;
-  }
-  return SOTER_SUCCESS;
+    if (EVP_DigestSignFinal(ctx->md_ctx, signature, signature_length) != 1) {
+        return SOTER_INVALID_SIGNATURE;
+    }
+    return SOTER_SUCCESS;
 }
 
 soter_status_t soter_sign_cleanup_ecdsa_none_pkcs8(soter_sign_ctx_t* ctx)
 {
-  if(!ctx){
-    return SOTER_INVALID_PARAMETER;
-  }
-  if(ctx->md_ctx){
-    EVP_MD_CTX_destroy(ctx->md_ctx);
-    ctx->md_ctx=NULL;
-  }
-  if(ctx->pkey_ctx){
-    EVP_PKEY *pkey = EVP_PKEY_CTX_get0_pkey(ctx->pkey_ctx);
-    if(pkey)EVP_PKEY_free(pkey);
-    EVP_PKEY_CTX_free(ctx->pkey_ctx);
-    ctx->pkey_ctx=NULL;
-  }
-  return SOTER_SUCCESS;
+    if (!ctx) {
+        return SOTER_INVALID_PARAMETER;
+    }
+    if (ctx->md_ctx) {
+        EVP_MD_CTX_destroy(ctx->md_ctx);
+        ctx->md_ctx = NULL;
+    }
+    if (ctx->pkey_ctx) {
+        EVP_PKEY* pkey = EVP_PKEY_CTX_get0_pkey(ctx->pkey_ctx);
+        if (pkey)
+            EVP_PKEY_free(pkey);
```

Comment:
when if's statement doesn't return then it not wrapped with braces if have one command. will be good to wrap all statements inside if/else statements